Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalMinBias_Output_cff.py

```diff
@@ -9,7 +9,10 @@
         SelectEvents = cms.vstring('pathALCARECOHcalCalMinBias')
     ),
     outputCommands = cms.untracked.vstring(
-	'keep *_gtDigisAlCaMB_*_*',
```

Comment:
Olga, one of the main users of the AlCaReco, could not run her codes using the old output commands. So our best bet is to use hltTriggerSummaryAOD and TriggerResults. We can drop the RAW information (which I consulted with Olga) and that is the reason for the increase in data volume. So I am updating this PR.